package.json

```diff
@@ -9,7 +9,7 @@
     "build": "tsc && vite build --base=/maputnik/",
     "build-desktop": "tsc && vite build --base=/ && cd desktop && make",
     "i18n:refresh": "i18next 'src/**/*.{ts,tsx,js,jsx}'",
-    "lint": "eslint ./src ./cypress --ext ts,tsx,js,jsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint ./src ./cypress --report-unused-disable-directives --max-warnings 0",
```

Comment:
Can we simplify this to just `eslint` and move all the flags to the config file?